### ISSUE TITLE
Namespace elements entered through wysiwyg input

### ISSUE THREAD

**Issue by zttodd** [indiana-university/rivet-source#218]
## Summary
Add a class or some other namespace to the container for content entered through a wysiwyg so all of the elements within are correctly styled.

## Describe the solution you'd like
```
<div class="rvt-wysiwyg">
    <ul>
        <li>Item 1<span></span></li>
        <li>Item 2</li>
        <li>Item 3</li>
    </ul>
    <br /><br />
    <p style="font-style: italic">Example text here</p>
    <br />
    <br />
</div>
```

The class `rvt-wysiwyg` will help style with any kind of extraneous or "junk" code that could get copied in from something like a Word doc.

Related to issue: #48

**Comment by illusivesunrae** [comment]
Merged into `2.0.0-dev` with PR #241.

**Comment by illusivesunrae** [comment]
Released with PR #294.
